pkg/schedule/operator_controller.go

```diff
@@ -424,14 +423,6 @@ func (oc *OperatorController) checkAddOperator(isPromoting bool, ops ...*operato
 		if op.SchedulerKind() == operator.OpAdmin || op.IsLeaveJointStateOperator() {
 			continue
 		}
-		if cl, ok := oc.cluster.(interface{ GetRegionLabeler() *labeler.RegionLabeler }); ok {
```

Comment:
It seems we need to use the way of pausing schedulers to prevent the range from being scheduled.

Reply:
I add the tests, prevent it in `coordinator`.